### PR TITLE
3794/group outcome suggestion routes into a controller

### PR DESCRIPTION
**Pull request by EVogel1999**
This PR pulls out the outcome api routes into the outcome suggestion controller and adds swagger docs to them.  Completes story [3794/group outcome suggestion routes into a controller](https://app.clubhouse.io/clarkcan/story/3794/group-outcome-suggestion-routes-into-a-controller).

### CODE REVIEW

File: src/modules/guideline-service/GuidelineServiceController.ts [hunk 1]
```diff
@@ -1,12 +1,95 @@
-import { Router } from "express";
+import { Router, Request } from "express";
+import proxy = require("express-http-proxy");
 import { Controller } from "../../interfaces/Controller";
 
+const OUTCOME_API = process.env.OUTCOME_API || 'localhost:3000';
```
Comment (pzalep1) [inline src/modules/guideline-service/GuidelineServiceController.ts:5]: I think outcome suggestion service runs on 9000.

**Review by pzalep1 — COMMENTED** [review]
I would check in the client to just double check my comment. I think this was part of the outcome release that we couldn't finish before we decided to pivot last June.

File: src/modules/guideline-service/GuidelineServiceController.ts (hunk 1 above)
Comment (EVogel1999) [reply src/modules/guideline-service/GuidelineServiceController.ts]: My .env says 7000, 9000 is utility service.  So I'm using 7000 instead.

Comment (pzalep1) [reply src/modules/guideline-service/GuidelineServiceController.ts]: Okay cool. I wasn't sure about the port number but 3000 sounded off.

**Review by pzalep1 — APPROVED** [review]
Looks good!

**Review by mfranz1 — APPROVED** [review]

**Review by jmulholland4014 — APPROVED** [review]

**Pull request merged**